packages/shorebird_cli/lib/src/shorebird_process.dart

```diff
@@ -14,6 +15,19 @@ final processRef = create(ShorebirdProcess.new);
 /// The [ShorebirdProcess] instance available in the current zone.
 ShorebirdProcess get process => read(processRef);
 
+/// Sanitizes the executable path on Windows.
+/// https://github.com/dart-lang/sdk/issues/37751
+String _sanitizeExecutablePath(String executable) {
+  if (executable.isEmpty) return executable;
+  if (!platform.isWindows) return executable;
+  if (executable.contains(' ') && !executable.contains('"')) {
```

Comment:
What about single quote?

Reply:
What do you mean?

Reply:
This appears to be trying to quote strings which have spaces but aren't already quoted.  Do we need to check for single-quote as a quote as well?

Reply:
I don't think so -- flutter_tool uses the same logic to sanitize executable paths on windows 🤷 
Going to leave it as is for now since it's consistent with flutter this way.